stock_by_warehouse/static/src/js/widget.js

```diff
@@ -42,5 +43,25 @@ var ShowPaymentLineWidget = form_common.extend({
     },
 });
 
+
+var ShowProductAvailability = form_common.extend({
```

Comment:
Modify the original one, I think we do not need to inherit here, due that the module is ours.

Reply:
Done :smile: